### DISCUSSION TITLE
how to change default button font weight and not all uppercase text

### DISCUSSION THREAD

**Discussion by ChrisCosentino** [saadeghi/daisyui#893] [Q&A] [answered]
On a daisy ui btn class how can you change the default behaviour in the theme daisyUi themes object in tailwind.config.js to set the default font weight and to remove the all uppercase text?

**Comment by saadeghi** [comment, marked as answer]
You can change the text transform in 3 ways:
- For all buttons: https://play.tailwindcss.com/Rchth3cCZG?file=css
- For a specific theme: https://play.tailwindcss.com/V18GqcTnaE?file=config
- For a specific button: https://play.tailwindcss.com/dgZ0zIkfz8

**Reply by azzamsa** [reply]
Why uppercase is the default in DaisyUI? Any reason for that?

**Reply by runarj** [reply]
It could be inspired from [Material 3's usage of capitalized text in buttons](https://material.io/components/buttons#anatomy).  Still, sentence case is also allowed in the spec.

**Reply by saadeghi** [reply]
It is indeed inspired by Material. To make action buttons look bold and draw attention.

**Reply by wuarmin** [reply]
@saadeghi Why doesn't the solution for all buttons look like this?:
```css
@tailwind base;
@tailwind components;
@tailwind utilities;

@layer components {
  .btn{
    text-transform: none;
  }
}
```
Thanks

**Reply by magiino** [reply]
Perhaps the only difference in overriding the definition for .btn in @layer components or outside is this:
https://tailwindcss.com/docs/adding-custom-styles#removing-unused-custom-css
